[{"tile": "AK_GlacierBay_4_2019", "error": "Expecting value: line 1 column 1 (char 0)"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/MO_WestCentral_1_2018/ept.json"], "error": "PDAL: readers.ept: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/MO_WestCentral_1_2018/ept.json\n\n"}, {"tile": "NE_Niobrara_Topobathy_2018", "error": "Expecting value: line 1 column 1 (char 0)"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/NJ_South_Jersey_FEMA_2018/ept.json"], "error": ""}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/NM_RioSanJose_TL_2016/ept.json"], "error": "PDAL: readers.ept: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/NM_RioSanJose_TL_2016/ept.json\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/NM_SouthEast_B5_2018/ept.json"], "error": "PDAL: readers.ept: Error reading tile: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/NM_SouthEast_B5_2018/ept-data/0-0-0-0.laz\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/OK_Panhandle_B2_2018/ept.json"], "error": ""}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/OR_NRCSUSGS_1_2019/ept.json"], "error": ""}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/PA_WesternPA_2_2019/ept.json"], "error": "PDAL: readers.ept: Error reading tile: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/PA_WesternPA_2_2019/ept-data/0-0-0-0.laz\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/SC_RichlandCo_2010/ept.json"], "error": "PDAL: readers.ept: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/SC_RichlandCo_2010/ept.json\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/SC_SaludaCo_2010/ept.json"], "error": ""}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/SD_EasternSD_1_2020/ept.json"], "error": "PDAL: readers.ept: Error reading tile: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/SD_EasternSD_1_2020/ept-data/2-2-3-1.laz\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/SD_NRCS_QSI_A2_2017/ept.json"], "error": "PDAL: readers.ept: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/SD_NRCS_QSI_A2_2017/ept.json\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/TX_Bell-Burnet-McLennan_2011/ept.json"], "error": "PDAL: readers.ept: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/TX_Bell-Burnet-McLennan_2011/ept.json\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/TX_Brazoria_2006/ept.json"], "error": ""}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/TX_DesertMountains_B2_2018/ept.json"], "error": ""}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_AK_POW_P2_2018_LAS_2019/ept.json"], "error": "PDAL: readers.ept: Error reading tile: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_AK_POW_P2_2018_LAS_2019/ept-data/2-1-3-2.laz\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_AR_Ouachita_B7_2016_LAS_2018/ept.json"], "error": "PDAL: readers.ept: Error reading tile: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_AR_Ouachita_B7_2016_LAS_2018/ept-data/1-1-1-0.laz\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_CA_SanDiego_2015_C17_1/ept.json"], "error": ""}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_NE_Hat_White_Holt_2016_LAS_2019/ept.json"], "error": "PDAL: readers.ept: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_NE_Hat_White_Holt_2016_LAS_2019/ept.json\n\n"}, {"error": "Could not create geometry because of errors while reading input."}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_NM_NRCS_Central_A1_2017_LAS_2018/ept.json"], "error": "PDAL: readers.ept: Error reading tile: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_NM_NRCS_Central_A1_2017_LAS_2018/ept-data/5-14-17-31.laz\n\n"}, {"error": "Could not create geometry because of errors while reading input."}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_NY_FEMA_R2_B1_2016_LAS_2018/ept.json"], "error": ""}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_NY_FEMA_R2_B2_2016_LAS_2018/ept.json"], "error": "PDAL: readers.ept: Error reading tile: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_NY_FEMA_R2_B2_2016_LAS_2018/ept-data/1-1-1-0.laz\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_OK_Area4_2011_LAS_2018/ept.json"], "error": "PDAL: readers.ept: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_OK_Area4_2011_LAS_2018/ept.json\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_SD_NRCS_QSI_A2_2017_LAS_2019/ept.json"], "error": "PDAL: readers.ept: Error reading tile: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_SD_NRCS_QSI_A2_2017_LAS_2019/ept-data/1-1-1-0.laz\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_TX_Lower_SanBernard_B4_2017_LAS_2019/ept.json"], "error": "PDAL: readers.ept: Error reading tile: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_TX_Lower_SanBernard_B4_2017_LAS_2019/ept-data/1-1-1-0.laz\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_TX_Panhandle_B6_2017_LAS_2019/ept.json"], "error": ""}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_TX_Panhandle_B8_2017_LAS_2019/ept.json"], "error": "PDAL: readers.ept: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_TX_Panhandle_B8_2017_LAS_2019/ept.json\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_TX_South_B5_2018_LAS_2019/ept.json"], "error": ""}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_TX_South_B6_2018_LAS_2019/ept.json"], "error": "PDAL: readers.ept: Error reading tile: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_TX_South_B6_2018_LAS_2019/ept-data/0-0-0-0.laz\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_UT_BearLake_Cache_2016_LAS_2018/ept.json"], "error": "PDAL: readers.ept: Error reading tile: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_UT_BearLake_Cache_2016_LAS_2018/ept-data/1-1-1-0.laz\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_UT_Washakie_Whites_Valley_2017_LAS_2017/ept.json"], "error": "PDAL: readers.ept: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_UT_Washakie_Whites_Valley_2017_LAS_2017/ept.json\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_VA_FEMA_R3_Southwest_B_2016_LAS_2018/ept.json"], "error": ""}, {"tile": "USGS_LPC_WA_Western_North_2016_LAS_2018", "error": "Expecting value: line 1 column 1 (char 0)"}, {"tile": "USGS_LPC_WA_Western_South_2016_LAS_2018", "error": "Expecting value: line 1 column 1 (char 0)"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/WA_FEMAHQ_B1_QL2_2018/ept.json"], "error": ""}, {"tile": "WA_Nisqually_Bathymetric_2020", "error": "Expecting value: line 1 column 1 (char 0)"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/WI_FondduLac_2018/ept.json"], "error": "PDAL: readers.ept: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/WI_FondduLac_2018/ept.json\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/WI_Oshkosh_3Rivers_FondDuLac_TL_2018/ept.json"], "error": "PDAL: readers.ept: Error reading tile: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/WI_Oshkosh_3Rivers_FondDuLac_TL_2018/ept-data/0-0-0-0.laz\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/WV_FEMAR3_Southcentral_B1_2018/ept.json"], "error": "PDAL: readers.ept: Error reading tile: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/WV_FEMAR3_Southcentral_B1_2018/ept-data/1-1-1-0.laz\n\n"}]